### ISSUE TITLE
Failing revolution tests in core's Travis run

### ISSUE THREAD

**Issue by kyleam** [datalad/datalad-revolution#92]
Core's [revolution run][0] is failing with

```
======================================================================
ERROR: Failure: ImportError (cannot import name 'RevolutionGitRepo')
----------------------------------------------------------------------
[...]
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad_revolution/tests/test_dataset.py", line 22, in <module>
    from ..annexrepo import RevolutionGitRepo as GitRepo
ImportError: cannot import name 'RevolutionGitRepo'
```

When we moved `Revolution{Git,Annex}Repo` to core, we added import aliases for the `Revolution*` classes.  But an alias wasn't added for *Git*Repo in *annex*repo.py because test_dataset.py was the only place that used this indirect import, and test_dataset.py was removed in the same commit the moved the classes to core.

So the core tests are trying to run tests from a module that doesn't exist in -revolution's tree for 0.8.0:

```
datalad-revolution% git describe HEAD
0.8.0
datalad-revolution% git ls-files | grep dataset
datalad_revolution/dataset.py
```

If I pip install the current -revolution, I see test_dataset.py:

```
% pip freeze | grep revol
datalad-revolution==0.8.0
% find ~/src/python/venvs/tmppip/lib/python3.5/site-packages/datalad_revolution | grep test_dataset
/home/kyle/src/python/venvs/tmppip/lib/python3.5/site-packages/datalad_revolution/tests/test_dataset.py
/home/kyle/src/python/venvs/tmppip/lib/python3.5/site-packages/datalad_revolution/tests/__pycache__/test_dataset.cpython-35.pyc
```

So the stale file must have made it into the pypi upload for 0.8.0.

[0]: https://travis-ci.org/datalad/datalad/jobs/502589309

**Comment by mih** [comment]
Thanks for the analysis -- that I can confirm. Not sure how that happened. The source tarball on pypi doesn't have the file, the wheel does.

**Comment by mih** [comment]
I re-released 0.8.0 as 0.8.1 on PyPi with the stale bits removed: https://github.com/datalad/datalad-revolution/commit/96d691ddd483ae539a3d144bb382ff380608b731

**Comment by kyleam** [comment]
Thanks!
